### PR TITLE
Updating copyright and MIT license

### PR DESCRIPTION
**Pull request by daineAMD**
Last PR got a little messy, and wanted to commit some changes without spamming with update emails, so here's a new PR.

### CODE REVIEW

File: clients/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # ########################################################################
-# Copyright 2016-2022 Advanced Micro Devices, Inc.
+# Copyright (C) 2016-2022 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (saadrahim) [inline clients/CMakeLists.txt:2]: nitpicking as no one wants to this again. I am not sure about spacing surrounding (C). I think you are right but should we confirm?

File: library/src/hipblas-config.cmake.in [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IM-
+# PLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNE-
+# CTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 # ########################################################################
```
Comment (amcamd) [inline library/src/hipblas-config.cmake.in:20]: Being picky, to be consistent you need a line with a single # between line 19 and 20

Comment (amcamd) [inline library/src/hipblas-config.cmake.in:20]: Being picky, to be consistent you need a line with a single # between line 19 and 20

File: install.sh [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IM-
+# PLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNE-
+# CTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+# ########################################################################
```
Comment (amcamd) [inline install.sh:22]: being picky, to be consistent, need a line with a single # between line 21 and 22

**Review by amcamd — APPROVED** [review]
Looks good. I will make similar changed in rocBLAS

File: clients/CMakeLists.txt (hunk 1 above)
Comment (daineAMD) [reply clients/CMakeLists.txt:2]: I asked in the ticket but haven't heard back, I'll go ahead with this merge.

**Comment by daineAMD** [comment]
Some CI failures due to docker failures and rocSOLVER CI failures. Built and passed tests on some instances, so will merge and keep an eye on sovler and hipblas CI.

**Pull request merged**